### PR TITLE
GameDB: add missing serials to GameDB

### PR DESCRIPTION
**Pull request by Mrlinkwii**
### Description of Changes

Adds missing serials to gameDB

### Rationale behind Changes

To have a more complete  GameDB
### Suggested Testing Steps

look at CI

### CODE REVIEW

File: bin/GameIndex.yaml [hunk 1]
```diff
@@ -18350,6 +18350,9 @@ SLPM-60106:
 SLPM-60107:
   name: "Golf Paradise"
   region: "NTSC-J"
+SLPM-60109:
+  name: " RIDGE RACER V [trail]"
```
Comment (IlDucci) [inline bin/GameIndex.yaml:18354]: Typo here. Should read [trial].

Comment (Mrlinkwii) [reply bin/GameIndex.yaml]: Done :+1:

File: bin/GameIndex.yaml [hunk 2]
```diff
@@ -18376,6 +18379,9 @@ SLPM-60265:
 SLPM-60273:
   name: "Zettai Zetsumei Toshi 2: Itetsuita Kioku-tachi [Trial B]"
   region: "NTSC-J"
+SLPM-60282:
+  name: "Lucky * Star RAvish Romance  [Trial]"
```
Comment (IlDucci) [inline bin/GameIndex.yaml:18383]: Possible typo here. Shouldn't have two spaces between Romance and [Trial].

File: bin/GameIndex.yaml [hunk 3]
```diff
@@ -18389,6 +18395,9 @@ SLPM-61011:
   region: "NTSC-J"
   speedHacks:
     InstantVU1SpeedHack: 0  # Fixes hang on FMV's when CDVD timing is accurate.
+SLPM-61018:
+  name: "Abarenbou Princess [Trial] "
```
Comment (IlDucci) [inline bin/GameIndex.yaml:18399]: Typo here. Shouldn't have an space after [Trial].

File: bin/GameIndex.yaml (hunk 2 above)
Comment (Mrlinkwii) [reply bin/GameIndex.yaml]: Done.

File: bin/GameIndex.yaml (hunk 3 above)
Comment (Mrlinkwii) [reply bin/GameIndex.yaml]: Done

**Comment by IlDucci** [comment]
LGTM from an English standpoint.

File: bin/GameIndex.yaml [hunk 4]
```diff
@@ -18350,6 +18350,9 @@ SLPM-60106:
 SLPM-60107:
   name: "Golf Paradise"
   region: "NTSC-J"
+SLPM-60109:
+  name: " RIDGE RACER V [Trial]"
```
Comment (lightningterror) [inline bin/GameIndex.yaml:18354]: Space after "

Comment (Mrlinkwii) [reply bin/GameIndex.yaml]: done.

File: bin/GameIndex.yaml [hunk 5]
```diff
@@ -18350,6 +18350,9 @@ SLPM-60106:
 SLPM-60107:
   name: "Golf Paradise"
   region: "NTSC-J"
+SLPM-60109:
+  name: "RIDGE RACER V [Trial]"
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:18354]: Ridge Racer V, no need to capitalise it. Example: http://www.thecoverproject.net/view.php?cover_id=4420

Comment (Mrlinkwii) [reply bin/GameIndex.yaml]: Done :+1:

**Pull request merged**